downstream/modules/platform/ref-projects-galaxy-support.adoc

```diff
@@ -0,0 +1,70 @@
+[id="ref-projects-galaxy-support"]
+
+= {Galaxy} support
+
+At the end of a Project update, {ControllerName} searches for the `requirements.yml` file in the `roles` directory, located at
+`<project-top-level-directory>/roles/requirements.yml`. 
+
+If this file is found, the following command automatically runs:
+
+[literal, options="nowrap" subs="+attributes"]
+----
+ansible-galaxy role install -r roles/requirements.yml -p <project-specific cache location>/requirements_roles -vvv
+----
+
+This file enables you to reference {Galaxy} roles or roles within other repositories which can be checked out in conjunction with your own
+project. 
+The addition of {Galaxy} support eliminates the need to create git submodules to achieve this result. 
+Given that SCM projects (along with roles/collections) are pulled into and executed from a private job environment, a `<private job directory>` specific to the project within `/tmp` is created by default. 
```

Comment:
Suggested re-write:
"Given that SCM projects, along with roles or collections, are pulled...